app/main.py

```diff
@@ -1,8 +1,56 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+        self.wash_ability = None
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float | int,
+                 clean_power: int,
+                 average_rating: float | int,
+                 count_of_ratings: int
+                 ) -> None:
+        self.distance_from_city_center = round(distance_from_city_center, 1)
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def wash_ability(self, car: Car) -> bool:
+        if car.clean_mark >= self.clean_power:
+            car.wash_ability = False
+            print(f"Can't wash {car.brand} with {car.clean_mark} mark")
```

Comment:
Do not create unnecessary prints or strings in the task, please follow the task description

Reply:
so , there is not allowed some creativity in code, ok(((